Modix.Bot/Modules/MessageContentPatternModule.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Threading.Tasks;
+using Discord.Commands;
+using Modix.Bot.Extensions;
+using Modix.Data.Models.Core;
+using Modix.Services.Blocklist;
+
+namespace Modix.Bot.Modules
+{
+    [Name("Patterns")]
+    [Summary("Use and maintain message content patterns.")]
+    [Group("pattern")]
+    [Alias("patterns")]
+    public class MessageContentPatternModule : ModuleBase
```

Comment:
Would it makes sense to have a "list" command? I can see it being very easy to forget what we have configured.